### ISSUE TITLE
graphviz

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#202]
The graphviz extension does not work for me. 

Trying the following code:

.. graph:: foo
    "bar" -- "baz";

I get an error:

WARNING: dot command 'dot' cannot be run (needed for graphviz output), check the graphviz_dot setting

```
Exception occurred:
  File "/Library/Frameworks/Python.framework/Versions/2.6/lib/python2.6/site-packages/Sphinx-0.6.2-py2.6.egg/sphinx/ext/graphviz.py", line 130, in render_dot_html
    fname, outfn = render_dot(self, code, options, 'png', prefix)
TypeError: 'NoneType' object is not iterable
The full traceback has been saved in /var/folders/+D/+DqyujUaFJyUdy7+OsFbKE+++yU/-Tmp-/sphinx-err-TmG5qs.log, if you want to report the issue to the author.
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/202
- Originally reported by: Anonymous
- Originally created at: 2009-06-18T14:01:52.370

**Comment by shimizukawa** [comment]
_From Dzhelil Rufat on 2009-06-19 00:02:45+00:00_

Apparently, I did not have the graphviz binary installed. 
I still think the program should have a clearer message about the binary not being installed, rather than throwing a TypeError which does not provide any guidance on how to fix the problem.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2009-07-09 18:26:00+00:00_

Should be fixed in <<cset ab4a507605461695fb7de20576c9db79c59fa985>>.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2013-03-31 13:57:11+00:00_

Removing version: 0.6.2 (automated comment)
